Source/Scene/Expression.js

```diff
@@ -43,6 +43,23 @@ define([
         }
     };
 
+    var unaryFunctions = {
+        abs : Math.abs,
+        sqrt : Math.sqrt,
+        cos : Math.cos,
+        sin : Math.sin,
+        tan : Math.tan,
+        acos : Math.acos,
+        asin : Math.asin,
+        atan : Math.atan,
+        radians : function(left) {
```

Comment:
Make sure CesiumMath is included correctly, check out https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Specs/Core/Cartesian4Spec.js for example.